### PR TITLE
Feat/mark point arc

### PR DESCRIPTION
**Pull request by skie1997**
[[中文版模板 / Chinese template](https://github.com/VisActor/vrender/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Release
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/VisActor/VChart/issues/2590

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: packages/vrender-components/src/marker/line.ts [hunk 1]
```diff
@@ -74,11 +76,32 @@ export class MarkLine extends MarkCommonLine<ILineGraphicAttribute, IMarkLineLab
...
-    return angle + (this.attribute.label.refAngle ?? 0);
+    const itemAngle = isPostiveXAxis(angle) ? angle : angle - Math.PI;
+    return itemAngle + (this.attribute.label.refAngle ?? 0);
   }
 
-  protected getTextStyle(position: IMarkLineLabelPosition) {
-    return DEFAULT_CARTESIAN_MARK_LINE_TEXT_STYLE_MAP[position];
+  protected getTextStyle(position: IMarkLineLabelPosition, labelAngle: number, autoRotate: boolean) {
+    // 垂直方向例外
+    if (Math.abs(labelAngle) === Math.PI / 2) {
```
Comment (xile611) [inline packages/vrender-components/src/marker/line.ts:85]: Math.PI / 2 是浮点数，不太适合用`===` 进行判断，

File: packages/vrender-components/src/marker/type.ts [hunk 2]
```diff
@@ -416,5 +417,27 @@ export type MarkPointAttrs = Omit<MarkerAttrs<MarkPointAnimationType>, 'labelSty
    */
   itemContent?: IItemContent;
 
+  /**
+   * 被标注的内容
+   */
+  targetItemContent?: {
```
Comment (kkxxkk2019) [inline packages/vrender-components/src/marker/type.ts:423]: 这个只能配置图形吧？命名为 targetSymbol 会不会合适一些？

File: packages/vrender-components/src/marker/type.ts [hunk 3]
```diff
@@ -416,5 +417,27 @@ export type MarkPointAttrs = Omit<MarkerAttrs<MarkPointAnimationType>, 'labelSty
...
+    /**
+     * 被标注内容与标记线间的间隙
+     * @default 0
+     */
+    margin?: number;
```
Comment (kkxxkk2019) [inline packages/vrender-components/src/marker/type.ts:428]: 这个是不是应该统一下，如果是两点之间的间隙，统一用 offset 吧

File: packages/vrender-components/src/marker/type.ts [hunk 4]
```diff
@@ -416,5 +417,27 @@ export type MarkPointAttrs = Omit<MarkerAttrs<MarkPointAnimationType>, 'labelSty
...
+     * 是否显示
+     * @default false
+     */
+    visible?: boolean;
+    /**
+     * 大小
+     * @default 20
+     */
+    size?: number;
+    style?: ISymbol;
```
Comment (kkxxkk2019) [inline packages/vrender-components/src/marker/type.ts:439]: 例子里，为什么 fill 和 stroke 写外面？

image

**Review by xile611 — APPROVED** [review]

**Pull request merged**